vm/src/builtins.rs

```diff
@@ -218,18 +218,30 @@ fn make_scope(
         }
         None => None,
     };
-
     let current_scope = vm.current_scope();
     let globals = match globals {
-        Some(dict) => dict.clone().downcast().unwrap(),
+        Some(dict) => {
+            let dict: PyDictRef = dict.clone().downcast().unwrap();
+            if !dict.contains_key("__builtins__", vm) {
+                let builtins_dict = match &vm.builtins.dict {
```

Comment:
Please change this match statement into an `.unwrap()` call. This will panic as well:
```rust
let builtins_dict = &vm.builtins.dict.unwrap().as_object();
```

Reply:
When I tried it, I got error like this:
```rust
error[E0507]: cannot move out of an `Rc`
   --> vm/src/builtins.rs:226:38
    |
226 |                 let builtins_dict = &vm.builtins.dict.unwrap().as_object();
    |                                      ^^^^^^^^^^^^^^^^ cannot move out of an `Rc`
```

Reply:
Maybe this works better:
```rust
let builtins_dict = vm.builtins.dict.as_ref().unwrap().as_object();
```

Reply:
Thanks! this code works well